conda-forge.yml

```diff
@@ -7,3 +7,5 @@ travis:
   secure: {BINSTAR_TOKEN: VmhwRIyGOSsYglStakb3SMVl/DWe3EvMKHqpbgtpUJu6qDfM0fHzCeFnCs8u0hPMF/G6uYu6sv8Ca5kIKbp0kLNluGeJCQpHJTwb5+wUXIZhokxeVatRTVOCfxAFXypO8phcxp6B/r4iNIpYBHZB8Y1iVMLDuOOhT0ZDX0qubakRM8ism95CP6R7NPsSQU1hydAXvWxIqqAkISRuJovUZSZF7tCUCHuquxvkbmXShHIMOpxynyVVZA6FKHgLjdvhfyDkOLvKcMGlIXxOvPS3rg8kwqG2VsyjBGjtrHDtaitCueUAOjqKTxWsbpbBc9HdK1sZDiWlBl+zSaMja2+WKZXUkQbcIxiyoeBNnYJCGK+eg44FtIfu65aDXkVrRoB+CbYOXAjWIQceIuWdqbB5fbBNC/FWlSQxFD4haBIb9hOYjCab8HzPz5Kr+6MkL4vYPTNqEGSkvMbFg24zznjNjowhkrwa3jBDOPejZrVaLNrbXXc6GEx5aqErgjoHyZWaKImGksrkKSTTKN4n9rzueBN2O0CPaRGxbeNfNtACSzcgQ2J+YYKaSFogJQzgcpV567kugICN5CDNWbp3NMU1pWFkWJF64FK5jMJvDmJE9mdJIFp517Btc4GR+K636gVm16Cl2SiUoRLatLODRcA/f9tQKzyDGQ0lyjVSfR0uE0U=}
 provider:
   win: azure
+  linux_aarch64: azure
```

Comment:
Any reason for using `azure` here rather the `default`.  On Azure Pipelines aarch64 is virtualized, defaults uses drone which uses aarch64 hardware.

Reply:
also can do if you like :-)
shall I redo ?